packages/react-app/src/components/quest.tsx

```diff
@@ -438,7 +464,8 @@ export default function Quest({
                     {(isPlayingQuest ||
                       questData.creatorAddress === walletAddress ||
                       (waitForClose && transaction?.type === TransactionType.QuestUnplay)) &&
-                      questData.maxPlayers !== undefined && ( // Make sure maxPlayers is set (play feature is available on this quest)
+                      questData.maxPlayers !== undefined &&
```

Comment:
le commentairre match pu yetait alligné avec cette ligne mais astheure on peut fair mieux  so tu peux l'enlever

```suggestion
                      questData.features.playableQuest &&
```